### PR TITLE
fix: detect new flex ads

### PR DESCRIPTION
**Pull request by cbotsikas**
#122 #123

### CODE REVIEW

File: src/hide_if_sponsored.js [hunk 1]
```diff
@@ -60,14 +58,14 @@ function getVisibleText(e) {
   }
   const children = e.querySelectorAll(":scope > *");
   if (children.length !== 0) {
-    if (e.style.display === "flex") {
```
Comment (tiratatp) [inline src/hide_if_sponsored.js:63]: let's create a `const style = window.getComputedStyle(e);` and use it for these

File: src/hide_if_sponsored.js [hunk 2]
```diff
@@ -122,6 +120,16 @@ function hideIfSponsored(possibleSponsoredTextQueries, e) {
...
+  if (e.dataset.pagelet === "FeedUnit_1") {
+    let preview = getVisibleText(e);
+    let index = preview.indexOf("·");
+    preview = preview.substring(0, index > 0 ? index : 50).trim() + "...";
+    console.info("ABfF:", `This is the second post and usually is an ad`, [
+      preview,
+      e,
+    ]);
+  }
+
```
Comment (tiratatp) [inline src/hide_if_sponsored.js]: Did you plan to add this so people can report a new bug easily?

**Review by tiratatp — CHANGES_REQUESTED** [review]
Two minor comments

File: src/hide_if_sponsored.js (hunk 1 above)
Comment (cbotsikas) [reply src/hide_if_sponsored.js:63]: done

File: src/hide_if_sponsored.js (hunk 2 above)
Comment (cbotsikas) [reply src/hide_if_sponsored.js]: yes, that was my idea. It's easier to instruct users how to verify (with the preview text) and copy (using console's context menu) the second element's dom.
Usually the second element is an ad, although i've seen some rare times that it was normal one and then the third was an ad 😕 
image

Comment (cbotsikas) [reply src/hide_if_sponsored.js]: added a comment in the code too

File: src/hide_if_sponsored.js (hunk 1 above)
Comment (tiratatp) [reply src/hide_if_sponsored.js:63]: Oh my bad. I thought these two are the same variable, but the second one is for the children's style.

**Review by tiratatp — APPROVED** [review]
lgtm

**Comment by tiratatp** [comment]
not sure why test was not passing but overriding.

**Pull request merged**